src/System.Private.ServiceModel/tools/SelfHostedWcfService/Program.cs

```diff
@@ -286,6 +286,10 @@ private static void Main()
             WebSocketHttpsDuplexTextBufferedTestServiceHost webSocketHttpsDuplexTextBufferedTestServiceHostServiceHost = new WebSocketHttpsDuplexTextBufferedTestServiceHost(typeof(WcfService.WSDuplexService), webSocketHttpsDuplexTextBufferedTestServiceHostbaseAddress);
             webSocketHttpsDuplexTextBufferedTestServiceHostServiceHost.Open();
 
+            Uri[] channelExtensibilityTestServiceHostbaseAddress = new Uri[] { new Uri(string.Format("{0}/ChannelExtensibility.svc", httpBaseAddress)) };
+            ChannelExtensibilityServiceHost channelExtensiblityTestServiceHostServiceHost = new ChannelExtensibilityServiceHost(typeof(WcfChannelExtensiblityService), channelExtensibilityTestServiceHostbaseAddress);
```

Comment:
I may have missed seeing it, but the web.config under tools\IISHostedWcfService needs to be updated to add an entry for the new servicehost.  I predict the Windows OuterLoops against IIS will fail without it.  Ignore this message if I missed seeing that file.